examples/webgl_loader_fbx_texture.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - FBX loader</title>
```

Comment:
Instead of adding a new example, please add the new asset to the existing https://threejs.org/examples/webgl_loader_fbx.

It already has a dropdown UI that allows to select for different assets.

Reply:
Thank you.
It looks like the title of the existing example has been set to Mixamo’s sample title.
https://github.com/mrdoob/three.js/blob/dev/examples/webgl_loader_fbx.html#L13
Would it be better to make the title change dynamically based on the pulldown selection?

Reply:
I don't think that is required.